src/node/mod.rs

```diff
@@ -313,6 +314,7 @@ impl Node {
     pub fn our_adults_sorted_by_distance_to(&self, name: &XorName) -> Vec<&P2pNode> {
         self.our_adults()
             .sorted_by(|lhs, rhs| name.cmp_distance(lhs.name(), rhs.name()))
+            .collect()
```

Comment:
Did this used to work without .collect() ?

Reply:
Yes, due to a change in the itertools crate. Previously `sorted_by` returned `Vec`, but now it return an iterator. But it's the `vec::IntoIter` iterator, so collecting it back into `Vec` is zero-cost.